browser/brave_shields/sources.gni

```diff
@@ -21,8 +21,9 @@ brave_browser_brave_shields_deps = [
   "//brave/browser:browser_process",
   "//brave/common",
   "//brave/components/brave_perf_predictor/browser",
-  "//brave/components/brave_shields/browser",
-  "//brave/components/brave_shields/common",
+  "//brave/components/brave_shields/content/browser",
+  "//brave/components/brave_shields/content/test:test_support",
```

Comment:
this shouldn't be here.

Reply:
I don't know why I added this. For some reason I thought it's needed here. Pushed fix.